grammar.js

```diff
@@ -242,6 +243,35 @@ module.exports = grammar({
       commaSep1(choice($.name, alias($._reserved_identifier, $.name), $.qualified_name))
     ),
 
+    enum_declaration: $ => prec.right(seq(
+      optional(field('attributes', $.attribute_list)),
+      keyword('enum'),
+      field('name', $.name),
+      optional(seq(':', $._type)),
+      optional($.class_interface_clause),
+      field('body', $.enum_declaration_list)
+    )),
+
+    enum_declaration_list: $ => seq(
+      '{',
+      repeat($._enum_member_declartaion),
+      '}',
+    ),
+
+    _enum_member_declartaion: $ => choice(
+      $.enum_case,
+      $.method_declaration,
+      $.use_declaration,
+    ),
+
+    enum_case: $ => seq(
+      optional(field('attributes', $.attribute_list)),
+      'case',
+      field('name', $.name),
+      optional(field('value', seq('=', choice($.string, $.integer)))),
```

Comment:
The `value` field should probably only be on the string/integer, and not on the equals sign.

Reply:
Fixed in the _bugfixes_ branch